internal/watcher/watcher.go

```diff
@@ -206,9 +199,14 @@ func (d Differential) WatchFn(path string) error {
 
 // NewDifferential returns initialized Differential.
 func NewDifferential(
-	ignoredExtensions, initialFilePaths []string,
+	allowedExtensions map[string]bool,
```

Comment:
1/5 that I liked the old API as the `map` is an implementation detail and a string slice is enough.

Reply:
Map usage changed to string slice: https://github.com/mattermost/mattermost-plugin-nagios/pull/31/commits/ef27e99c2f5e1082e5c5de658675f9d2d2f0e4ee